.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -7,12 +7,28 @@ assignees: ''
 
 ---
 
-*Before creating a feature request, please search existing feature requests to see if you find a similar one. If there is a similar feature request please up-vote it and/or add your comments to it instead*
+<!--
+Thank you for opening this feature request! You are helping make eksctl a better project :)
+In order to help us process this request faster, please provide as much detail as possible.
+
+Before creating a feature request, please search existing issues to see if this request has already been made. If there is a similar feature request please up-vote it and/or add your comments to it instead.
+
+For more information on how to contribute to eksctl, please refer to our contributing docs.
+https://github.com/weaveworks/eksctl/blob/master/CONTRIBUTING.md#opening-issues
+-->
 
 **Why do you want this feature?**
-A clear and concise description of why you want this feature. Include use-cases and any specific constraints you may have.
+<!--
+A clear and concise description of why you want this feature.
+What is it you are trying to accomplish?
+Include use-cases and any specific constraints you may have.
+-->
 
 **What feature/behavior/change do you want?**
```

Comment:
Can the order be changed so we ask what they want first, and then why afterwards?

Reply:
yep 👍